pom.xml

```diff
@@ -53,9 +49,8 @@
     <sonar.exclusions>target/generated-sources/**/*</sonar.exclusions>
     <gitRepositoryName>sonar-scanner-jenkins</gitRepositoryName>
     <version.artifactory.plugin>2.6.1</version.artifactory.plugin>
-    <jenkins.version>2.361.1</jenkins.version>
+    <jenkins.version>2.387.3</jenkins.version>
```

Comment:
the current recommended version:
https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#changing-the-minimum-required-version